ydb/core/tx/schemeshard/ut_helpers/helpers.cpp

```diff
@@ -2688,6 +2689,21 @@ namespace NSchemeShardUT_Private {
         return result;
     }
 
+    ui32 CountRows(TTestActorRuntime& runtime, ui64 schemeshardId, const TString& table)
+    {
+        auto tableDesc = DescribePath(runtime, schemeshardId, table, true, true, true);
```

Comment:
```suggestion
        auto tableDesc = DescribePath(runtime, schemeshardId, table, true, false, true);
```

Достаточно так. Границы шардов не нужны.